src/routes/followers.route.ts

```diff
@@ -0,0 +1,40 @@
+import express, { Request, Response } from "express";
+
+import { followersSchema } from "../helpers/validation";
+import { isAuthorized } from "../middlewares/isAuthorized.middlewares";
+import { validation } from "../middlewares/yup.middlewares";
+import {
+  addFollower,
+  deleteFollower,
+  getFollowers,
+} from "../services/followers.service";
+
+const followersRouter = express.Router();
+
+followersRouter
+  .route("/")
+  .get(isAuthorized, async (req: Request, res: Response) => {
+    const result = await getFollowers(req.query);
```

Comment:
```suggestion
    const result = await getFollowers(new PaginationParameters(req));
```
 tweet route for example